src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -53,126 +58,155 @@ const CustomizeOptions = ({
   selectedImage,
   updatedDate,
   url,
-}) => (
-  <div className="modal-body-blocksmith">
-    <ImageBlockPreview className="my-4" block={block} />
-    {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
-      <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
-    )}
-    <FormGroup>
-      <ImageAssetPicker
-        multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
-        isImageSelected={!!selectedImage}
-        isModalOpen={isAssetPickerModalOpen}
-        isDisabled={false}
-        label="Select an image"
-        node={node}
-        onClearImage={handleClearImage}
-        onSelectImage={handleSelectImage}
-        onToggleAssetPickerModal={handleToggleAssetPickerModal}
-      />
-    </FormGroup>
-    <div style={{ maxWidth: '350px', margin: '0 auto' }}>
+}) => {
+  const hasThemeableMixin = memoize(() => node.schema().hasMixin('triniti:common:mixin:themeable'));
+
+  return (
+    <div className="modal-body-blocksmith">
+      <ImageBlockPreview className="my-4" block={block} />
+      {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
+        <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
+      )}
       <FormGroup>
-        <Label>Aspect Ratio</Label>
-        <Select
-          onChange={handleChangeAspectRatio}
-          value={!aspectRatio.value ? null : {
-            label: aspectRatio.value.replace('by', ' by '),
-            value: aspectRatio.value,
-          }}
-          options={aspectRatioOptions}
+        <ImageAssetPicker
+          multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
+          isImageSelected={!!selectedImage}
+          isModalOpen={isAssetPickerModalOpen}
+          isDisabled={false}
+          label="Select an image"
+          node={node}
+          onClearImage={handleClearImage}
+          onSelectImage={handleSelectImage}
+          onToggleAssetPickerModal={handleToggleAssetPickerModal}
         />
       </FormGroup>
-      <FormGroup className="d-flex mb-2">
-        <FormGroup check>
-          <Label check>
-            <Checkbox size="sd" id="isNsfw" checked={isNsfw} onChange={handleChangeCheckbox} />
-            is nsfw
-          </Label>
+      <div style={{ maxWidth: '350px', margin: '0 auto' }}>
```

Comment:
@jalopez13 do we need this explicit styling here?  seems like we should be using class or admin-ui components for this.  @mgollnick what say you?

Reply:
@gdbrown 

abstracted styling into styles.scss.